html-forms/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="styles.css" href="styles.css">
+    <title>HTML-Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form method="post">
+      <label for="email">Email</label><br />
+      <input type="email" id="email" name="user_email" placeholder="example@email.com"/><br />
+
+      <label for="interest">Why is coding cool?</label><br />
+      <select name="code-is" id="interest">
+        <option value="choice0">Select an option</option>
+        <option value="choice1">Because it's interesting</option>
+        <option value="choice2">Because it is fun</option>
+        <option value="choice3">Because it is powerful</option>
+      </select><br />
+
+      <label for="fav">Favorite language (choose one)</label><br />
+      <label for="html"><input type="radio" id="html" name="lang"/>HTML</label>
+      <label for="css"><input type="radio" id="css" name="lang" />CSS</label>
+      <label for="js"><input type="radio" id="js" name="lang"/>JavaScript</label><br />
+
+      <p><label for="info">Would you like to sign up for more information?</label></p>
+
+      <label for="sign-up"><input type="checkbox" id="sign-up"/>Sign me up!</label><br />
+      <label>Comments</label><br />
+      <textarea></textarea><br />
```

Comment:
let's add the `for` attribute to the `label` and have the matching `id` attribute on the `textarea` so that they are linked